Backend/routes/auth.js

```diff
@@ -0,0 +1,14 @@
+const ans = require('../db/api_call');
+const hashString = require('../env/variable');
+
+
+var authreq = async (req, res, next) => {
+
+    if (req.headers.authorization === hashString) {
+        next()

```

Comment:
check for semicolons after function calls (and on line endings where appropriate)  throughout app
ex: 
next();